platform/ui/src/views/StudyList/components/StudyListTable.js

```diff
@@ -9,7 +11,29 @@ import InstancesActive from '../../../assets/icons/instances-active.svg';
 import InstancesInactive from '../../../assets/icons/instances-inactive.svg';
 import LaunchInfo from '../../../assets/icons/launch-info.svg';
 
-const TableRow = () => {
+const getStudyModalities = series => {
+  const modalities = series.reduce((acc, item) => {
+    const { Modality } = item;
+    if (acc.includes(Modality)) {
+      return acc;
+    }
+
+    acc.push(Modality);
+    return acc;
+  }, []);
+
+  return modalities.join('/');
+};
+
+const getInstances = series => {
```

Comment:
Same here. All data transformations from a "domain" model to a "view model" should happen before components touch data.